snuba/replacer.py

```diff
@@ -267,6 +281,9 @@ def __init__(self, storage: WritableTableStorage, metrics: MetricsBackend) -> No
         self.__sharded_pool = RoundRobinConnectionPool(self.__storage.get_cluster())
         self.__rate_limiter = RateLimiter("replacements")
 
+        self.__prolonged_offset = float("-inf")
```

Comment:
You could just use the value "-1" here like you do when bypassing checks.

Reply:
`float("-inf")` is necessary to tell the replacer to look in Redis. `-1` means we know there is nothing in Redis. This allows us to only read from Redis once and then maintain the state locally as well as in Redis.

Reply:
Could we renamed this as, "last_offset_processed"?

Reply:
I've renamed it but I feel it may be a bit misleading, it's the last offset processed _that took more than 2 min to process_ which is why I went with "prolonged" offset initially.